src/main/kotlin/net/ccbluex/liquidbounce/event/Event.kt

```diff
@@ -51,3 +55,5 @@ open class CancellableEvent : Event() {
 enum class EventState(val stateName: String) {
     PRE("PRE"), POST("POST")
 }
+
+fun KClass<out Event>.name(): String = this.findAnnotation<Nameable>()!!.name
```

Comment:
Remove this, because after changing the lookup to hashmap it is not used anymore.